lib/views/home.dart

```diff
@@ -0,0 +1,296 @@
+              ),
+              child: SafeArea(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    const SizedBox(height: 40),
+                    Padding(
+                      padding: const EdgeInsets.symmetric(horizontal: 16),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          Text(
+                            "Recently Played",
+                            style: Theme.of(context).textTheme.headline6,
+                          ),
+                          Row(
+                            children: const [
+                              Icon(Icons.history),
+                              SizedBox(width: 16),
+                              Icon(Icons.settings),
+                            ],
+                          )
+                        ],
+                      ),
+                    ),
+                    SingleChildScrollView(
+                      scrollDirection: Axis.horizontal,
+                      physics: const BouncingScrollPhysics(),
+                      padding: const EdgeInsets.all(16),
+                      child: Row(
+                        children: const [
+                          AlbumCard(
+                            label: "Best Mode",
+                            image: AssetImage("assets/album7.jpg"),
+                          ),
+                          SizedBox(width: 16),
+                          AlbumCard(
+                            label: "Motivation Mix",
+                            image: AssetImage("assets/album2.jpg"),
+                          ),
+                          SizedBox(width: 16),
+                          AlbumCard(
+                            label: "Top 50-Global",
+                            image: AssetImage("assets/top50.jpg"),
+                          ),
+                          SizedBox(width: 16),
+                          AlbumCard(
+                            label: "Power Gaming",
+                            image: AssetImage("assets/album1.jpg"),
+                          ),
+                          SizedBox(width: 16),
+                          AlbumCard(
+                            label: "Top songs - Global",
+                            image: AssetImage("assets/album9.jpg"),
+                          ),
+                        ],
+                      ),
+                    ),
+                    const SizedBox(height: 32),
+                    Padding(
+                      padding: const EdgeInsets.all(16.0),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.stretch,
+                        children: [
+                          Text(
+                            "Good evening",
```

Comment:
Ideally, you can create a list of songs/albums you want to display (as a plain dart data structure), and then use `for` loop or `list.map` to project that data to widgets that display them